lib/models/organization_member_model.dart

```diff
@@ -1,30 +1,54 @@
+class OrganizationMemberModelList {
+  final List<OrganizationMemberModel> data;
+  final int? status;
+  final String? message;
+
+  OrganizationMemberModelList(
+      {required this.data, required this.status, required this.message});
+
+  factory OrganizationMemberModelList.fromJson(Map<String, dynamic> json) {
+    // List<OrganizationMemberModel> organizationMembersList =
```

Comment:
Please remove commented codes that are not in use.

Reply:
okay